flocker/control/_model.py

```diff
@@ -110,23 +111,33 @@ def pmap_field(
     :param initial: An initial value for the field.  This will first be coerced
         using the field's factory.  If not given, the initial value is an empty
         map.
+    :param factory: A factory used to convert input arguments to the stored
+        value whenever it is set. Note that this will be composed with the
+        constructor for the ``CheckedPMap`` class constructed for this field.
```

Comment:
Yea +1, just like factory for the normal `field`. Maybe not the best... but fit the existing style.